### ISSUE TITLE
Error in running server3

### ISSUE THREAD

**Issue by paragsahu03** [insidesherpa/JPMC-tech-task-2-PY3#314]
![git](https://user-images.githubusercontent.com/42367866/83306886-09c8da80-a221-11ea-8b87-7489b0bb0592.jpg)

**Comment by Aman22sharma** [comment]
@paragsahu03 yes I am also facing this same error. Have you resolved this error?
